src/components/GetActivities.js

```diff
@@ -0,0 +1,44 @@
+import eventList from "../data/events.json"
+import Title from "./Title"
+import { FaTwitter, FaDiscord } from "react-icons/fa"
+
+const events = eventList.events
+const styles = "transition-all ease-in cursor-pointer"
+
+const GetActivities = () => {
+  return (
+    <div className="flex flex-col items-center">
+      <Title heading="Activities" />
+      <section className="flex w-full flex-wrap items-center justify-center p-4 sm:p-10">
+        {events.map((event, idx) => {
+          return (
+            event.isActive && (
+              <div key={idx} className='hov-bg-theme m-5 h-[450px] rounded-xl bg-[#f1f5f9] p-5 text-black shadow-md transition-all ease-out xl:w-[380px]'>
+                <div>
+                  <h2 className="title mt-5 text-2xl">{event.name}</h2>
+                  <h3 className="text-xl text-right font-light">By {event.organizer}</h3>
+                  <h4 className="text-xl font-semibold">{event.date}</h4>
+                  <img src={event?.image} alt={event.name} className='max-h-[20rem] rounded-md object-contain' />
```

Comment:
Please use the Image tag from NextJS

Reply:
The width is 272 pixels and the height is what 20rem in tailwind is translated to pixels. @AryanBabber.